rpc/legacyrpc/methods.go

```diff
@@ -477,6 +479,37 @@ func createMultiSig(s *Server, icmd interface{}) (interface{}, error) {
 	}, nil
 }
 
+func createVotingAccount(s *Server, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrjson.CreateVotingAccountCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, errUnloadedWallet
+	}
+
+	name := cmd.Name
+	if len(name) == 0 {
+		return nil, rpcErrorf(dcrjson.ErrRPCInvalidParameter, "empty name")
+	}
+
+	key := cmd.PubKey
+	if len(key) == 0 {
+		return nil, rpcErrorf(dcrjson.ErrRPCInvalidParameter, "empty pubkey")
```

Comment:
parse/decode the xpub here using [hdkeychain.NewKeyFromString](https://godoc.org/github.com/decred/dcrd/hdkeychain#NewKeyFromString)